### ISSUE TITLE
Is all print advertising a violation of collective privacy?

### ISSUE THREAD

**Issue by jyasskin** [w3ctag/privacy-principles#235]
@michaelkleber points out that https://w3ctag.github.io/privacy-principles/#collective includes

> Group-level [data processing](https://w3ctag.github.io/privacy-principles/#dfn-process) may impact populations or individuals, including in ways that [people](https://w3ctag.github.io/privacy-principles/#dfn-individual) could not control even under the optimistic assumptions of [consent](https://w3ctag.github.io/privacy-principles/#dfn-opt-in). For example, based on group-level analysis, a company may know that site.example is predominantly visited by [people](https://w3ctag.github.io/privacy-principles/#dfn-individual) of a given race or gender, and decide not to run its job ads there. Visitors to that page are implicitly having their [data](https://w3ctag.github.io/privacy-principles/#dfn-data) processed in [inappropriate](https://w3ctag.github.io/privacy-principles/#dfn-inappropriately) ways, with no way to discover the discrimination or seek relief ([[Relational-Governance](https://w3ctag.github.io/privacy-principles/#bib-relational-governance)]).

But advertising in paper magazines and newspapers is based on exactly this sort of group-level analysis, and the sort of racial discrimination we're worried about is probably easier to avoid on the web than in print, where advertisers can target some details of individual people instead of just those people's likely race or gender.

We should make sure not to imply that this is a web-only problem, or to hold the web to a significantly higher bar than we hold print advertising in practice.

**Comment by dmarti** [comment]
Related: https://github.com/patcg/docs-and-reports/issues/36

In general, print advertising is accountable several ways. First, a regulator, journalist or researcher can buy newsstand copies and look at the ads.

Second, the advertiser and/or agency gets stuck with piles of insertion orders and tearsheets. If there is an investigation into discriminatory practices, regulators  would likely be able to see those records.

In print, discrimination is possible but obvious.

**Comment by jyasskin** [comment]
[Investigations into discriminatory practices find evidence of them online too](https://www.propublica.org/article/facebook-advertising-discrimination-housing-race-sex-national-origin), so it's not like there's "no way to discover the discrimination". It also takes [actual research](https://academic.oup.com/her/article-abstract/6/4/487/568808) to show that offline advertising is discriminatory, so it's not right to say that offline discrimination is "obvious".

**Comment by dmarti** [comment]
Good point. Instead of "no way to discover" it might make sense to change it to "no way to discover without doing costly, time-consuming research and violating terms of service".

The study that found this housing ad discrimination was a major, long-term effort, including actually making fake accounts in violation of platform ToS, and buying advertising using those accounts.

That's a good link to research on offline advertising (counting billboards in neighborhoods and comparing to census data). This research was quantifying an obvious pattern of discrimination that was already noticeable--and the researchers did not have to buy billboard advertising or violate any ToS in order to do it.

**Comment by darobin** [comment]
Right, I think it's important to capture here that print advertising is more accountable because many can observe at low cost, whereas targeted advertising makes accountability difficult and costly. That's worth pointing out because it also points at ways in which we could support better accountability. One way is to require machine-readable metadata in ads. That way we can use tools like Citizen Browser or Rally to produce large-scale reports on advertising practices.

I think we could capture that discussion in there, no?

**Comment by npdoty** [comment]
There are definitely many offline violations of privacy, and I don't think we would expect or want principles for privacy on the Web to imply that everything done in offline contexts is always appropriate.

On collective privacy, I don't know that we have a particular answer or a bright line on what is acceptable or what is something that we can practically affect. I think privacy of small groups is more important and easier to address than privacy of large groups, for example, even though there are important cases of inappropriate processing that only affect individuals because of their membership in large groups.

**Comment by dmarti** [comment]
@darobin The main problem for cooperative ad analysis tools is [out of band actions by parties that deliver deceptive ads](https://algorithmwatch.org/en/instagram-research-shut-down-by-facebook/) not missing metadata.

A general principle, which may or may not be out of scope for this document, is that users have a shared interest in becoming better informed about how their information is used, user agents should facilitate collective action like research to help serve this interest, and other parties should not interfere.

**Comment by wseltzer** [comment]
Several differences between print and digital seem relevant:
* transparency and observability (problems of Terms of Service in particular)
* the nature of feedback loops (response measurement, speed, and ability to tweak)
* the ability to individualize and subdivide groups (economically, at scale)

Advertising in the print edition of Furries Weekly to attract or exclude a particular class of readers might raise a discrimination problem, but not one specifically tied to privacy (at least not until they add "Call and ask for extension FUR").

**Comment by dmarti** [comment]
@wseltzer  Another relevant difference between print and digital is that there is no obfuscation layer between print advertiser and print publisher. No ad agency is going to tell the client, "we're going to run your print ad in either _People_ magazine or in _Furries Weekly_, but we won't tell you which one" -- but that's a pretty common practice in digital ads.

If the advertiser is investigated for discriminatory practices (such as targeting of rental housing ads by protected group) then they will have tearsheets and insertion orders for their print ads, but possibly only a minimal, obfuscated report on their digital ads, which hides discriminatory placements (and other problems).

**Comment by michaelkleber** [comment]
Hi folks!  I agree that there are vast differences between print and online advertising.  But even so, I am surprised by the current text, which implies that a decision like "Don't show my ad on ebony.com" is a violation of the privacy of people who visit that website.  Jeffrey's #248 entirely reasonably puts that on the same footing as "Don't run my ad in Ebony magazine", but I don't think I buy the implication in either the print or online case.

Maybe I am out of touch with the collective privacy thinking, in which case I'm happy to be educated.  But my limited understanding of the Viljoen article doesn't support this point of view.

I'm happy to see the focus on discoverability and accountability in the above discussion.  It seems to me that "show my ad wherever an ML algorithm feels like showing it" is the inherently risky decision, from that point of view, because potential bias hidden inside the model is much harder to discover than the ebony.com example.  Is there a way to focus the Collective Governance example on that instead?

**Comment by jyasskin** [comment]
The resolution of this issue was to remove the problematic example, and https://github.com/w3ctag/privacy-principles/issues/251 tracks adding a better example.
